README.md

```diff
@@ -20,26 +20,31 @@ Or install it yourself as:
 
 ## Usage
 
-### Sinatra instructions
-#### App setup
-* Add to Gemfile as well as listen 3.1.1
-* Require in server file (i.e. config.ru)
```

Comment:
As it stands, the server side just sends up the ruby and doesn't get involved in the pipeline.

However! A future improvement will be to put it into config.ru _instead_ of starting it by hand, so that when you run rackup it will start the websocket-based asset reloader server for you automatically, maybe via a fork().